QUEST.tsv

```diff
@@ -3292,11 +3292,11 @@ QUEST_20151224_003299
 QUEST_20151224_003300	
 QUEST_20151224_003301	
 QUEST_20151224_003302	
-QUEST_20151224_003303	This is the catastrophe that already fell upon us.{nl}But an ever greater calamity is just beginning.{nl}
-QUEST_20151224_003304	But... Where could the goddess of destiny be?{nl}And the other goddesses...?{nl}
-QUEST_20151224_003305	See?{nl}No goddesses can protect you from death.{nl}
-QUEST_20151224_003306	That's why the Revelator...{nl}I close my eyes and ears...
-QUEST_20151224_003307	Just run away like everyone else did.
+QUEST_20151224_003303	This is the catastrophe that already fell upon us...{nl}But an ever greater calamity is just the beginning.{nl}
```

Comment:
But an ever greater calamity is just the beginning.

But an **even** greater calamity is **just beginning**.